### DISCUSSION TITLE
URL Scheme to sync on demand

### DISCUSSION THREAD

**Discussion by sj7trunks** [MobiusSync/MobiusSync#131] [Ideas]
Can we get an option to sync via shortcut? As part of my workflow I launch mobius sync before jumping into Logseq. I use Logseq as a todo app and I need to sync more often than möbius syncs. Ideally you should be able to sync in the background via a shortcut before opening your application.

https://developer.apple.com/documentation/xcode/defining-a-custom-url-scheme-for-your-app

**Comment by jsamuel1** [comment]
This would also be useful for apps like Obsidian, when syncing their folder via syncthing -- so that they could trigger a sync on startup.  (eg. an addin such as https://github.com/LBF38/obsidian-syncthing-integration/ could call the URL scheme to start a sync or to launch into the Mobius GUI from Obsidian)

**Reply by l1mey112** [reply]
It would be nice if this existed, otherwise I am moving to SyncTrain.

**Reply by SimonPickup** [reply]
This is logged as an issue here: #136
